src/Utils/samples.jl

```diff
@@ -139,54 +144,52 @@ original set ``X`` in all ``n`` dimensions. The function rejects a drawn sample
 struct RejectionSampler{D} <: AbstractSampler
     distribution::D
     tight::Bool
+    bound::Number
```

Comment:
i suggest to call this differently, eg. `maxiter` .
also i think that you forgot adding the new field to the docstring.

Reply:
Right. Done.
I suggest that we first merge #2563.